src/app/main/connection-window/connection-window.component.ts

```diff
@@ -54,6 +55,18 @@ export class ConnectionWindowComponent extends SubscriptionHandler implements On
     }
   }
 
+  /*
+    AfterViewInit, a lifecycle hook that is called after Angular has fully initialized a component's view.
+    forces the login if auto_login is detected.
+  */
+  ngAfterViewInit(): void {
+    const auto_login = localStorage.getItem('auto_login') === 'true';
+    if (auto_login) {
+      let el: HTMLElement = this.myDiv.nativeElement;
+      el.click();
```

Comment:
```suggestion
      this.myDiv.nativeElement.click();
```

you can avoid to create a variable